apps/accounts/filters.py

```diff
@@ -150,7 +150,7 @@ class Meta:
         fields = ["id", "username"]
 
 
-class SecretRecordMixin:
+class SecretRecordMixin(drf_filters.FilterSet):
     asset_name = drf_filters.CharFilter(
         field_name="asset__name", lookup_expr="icontains"
     )
```

Comment:
There is no specific coding issue mentioned above. The provided code seems to be in a Django environment.

However, I can offer some advice on potential improvements:

- Use of 'Asset' instead of an object-level queryset when filtering with drf_filters would lead us towards having a cleaner approach that could help to avoid circular references if the `asset` attribute was defined outside this module.
   
  - Class name suggestion: SecretRecordQuerySet

This way, we should not have reference cycles like:

```
class Asset(models.Model): pass
@SecretRecordMixin
class SecretRecord(Questionnaire):
    asset: Asset = models.ForeignKey(Asset, related_name='secrets', ...)
```